src/mpsc/errors.rs

```diff
@@ -1,6 +1,26 @@
 //! Errors returned by channels.
 use core::fmt;
 
+/// Error returned by the [`Sender::send_timeout`] or [`Sender::send_ref_timeout`] 
+/// (and [`StaticSender::send_timeout`]/[`StaticSender::send_ref_timeout`]) methods.
```

Comment:
since these are only present on the blocking channel, let's add
```suggestion
/// (and [`StaticSender::send_timeout`]/[`StaticSender::send_ref_timeout`]) methods
/// (blocking only).
```